content/entry-hash/index.md

```diff
@@ -0,0 +1,176 @@
+---
+rfc:
+start_date: 2018-08-02
+pr:
+status: draft
+---
+
+# Entry hash
+
+## Summary
+
+This RFC proposes an algorithm to hash entries without relying in JSON or in
+the attribute names.
```

Comment:
Fair question; the problem is that the data in the entry is mixed with the serialisation of the structure of the entry and, because the hash is for integrity purposes, if I decide to change how I represent the entry, but not the entry itself, I'm changing its identity. I should explain this in the RFC 👍